modules/logging-create-loki-cr-console.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-create-loki-cr-console_{context}"]
+= Creating a LokiStack custom resource by using the web console
+
+You can create a `LokiStack` custom resource (CR) by using the {product-title} web console.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+* You installed the {loki-op}.
+
+.Procedure
+
+. Go to the *Operators* -> *Installed Operators* page. Click the *All instances* tab.
+
+. From the *Create new* drop-down list, select *LokiStack*.
+
+. Select *YAML view*, and then use the following template to create a `LokiStack` CR:
+// tag::pre-5.9[]
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <2>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: '2022-06-01'
+    secret:
+      name: logging-loki-s3 # <3>
+      type: s3 # <4>
+      credentialMode: static #
```

Comment:
`credentialMode` should not be specified on LokiStack CR on pre 5.9 Loki installations.